2.3v2.c

```diff
@@ -0,0 +1,53 @@
+#include <stdio.h>
+
+
+//�������� ���������� ��������� � ��������.
+
+
+#define range 2001
+
+int sortOfNumbers() {

```

Comment:
```suggestion
int countiongSort(int* arrayForSort, int size) {
```
1) Именование
2) Функция должна принять считанный в main массив